src/Events/Custom_Tables/V1/Models/Builder.php

```diff
@@ -533,8 +525,9 @@ public function update( array $data = null ) {
 
 		// If we have a cache, let's clear it.
 		$model->flush_cache();
+		$query_result = $this->execute_queries ? $this->query( $SQL ) : false;
```

Comment:
```suggestion
		return $this->execute_queries ? $this->query( $SQL ) : false;
```
(and remove the return below)